modules/custom/az_paragraphs/az_paragraphs_views_side_by_side/az_paragraphs_views_side_by_side.info.yml

```
@@ -0,0 +1,11 @@
+name: 'Quickstart Paragraphs - Views Side-by-Side'
+type: module
+package: 'The University of Arizona - Experimental'
+description: 'Provides a paragraph type to show two views side-by-side on desktop screens.'
+core_version_requirement: ^9 || ^10
+dependencies:
+  - az_paragraphs:az_paragraphs
+  - drupal:field
+  - drupal:views
+  - paragraphs:paragraphs
+  - viewsreference:viewsreference
```

I _think_ you could potentially replace all of these with just `az_paragraphs_view`.
```suggestion
  - az_paragraphs:az_paragraphs_view
```